src/main/scala/io/citrine/lolo/stats/StatsUtils.scala

```diff
@@ -58,4 +62,7 @@ object StatsUtils {
     if (sigma2X == 0 || sigma2Y == 0) return 0.0
     covariance(X, Y, Some(actualWeights)) / math.sqrt(sigma2X * sigma2Y)
   }
+
+  def breezeRandBasis(rng: Random): RandBasis =
```

Comment:
ya know what I just realized. We have roughly this same function within `TestUtils.getBreezeRandBasis`. 

if its easy we should delete that one, and do a find/replace to use this one.